python/triton/runtime/autotuner.py

```diff
@@ -172,10 +172,13 @@ def check_disk_cache(self, tuning_key, configs, bench_fn):
         from triton._C.libtriton import get_cache_invalidating_env_vars
         from triton.compiler.compiler import make_backend, triton_key
         from triton.runtime.cache import get_cache_manager
+        from triton.runtime.interpreter import InterpretedFunction
         from triton.runtime.jit import JITFunction
 
         fn = self.fn
         while not isinstance(fn, JITFunction):
+            if isinstance(fn, InterpretedFunction):
+                return bench_fn()
```

Comment:
Could we instead override `cache_results` to false when the interpreter is enabled globally? That would remove runtime overhead when calling a function with a new config.

Reply:
It also occurs to me that autotuning while interpreting is a strange thing to do, but I guess it's benign.

Reply:
Sure, that's a good idea

Reply:
Done - this is a very trivial fix now :-D